opensrp-giz-malawi/src/main/java/org/smartregister/giz/fragment/ChildRegisterFragment.java

```diff
@@ -37,9 +38,26 @@ public void setAdvancedSearchFormData(HashMap<String, String> advancedSearchForm
         //do nothing
     }
 
+
+    @Override
+    protected void toggleFilterSelection() {
+        if (filterSection != null) {
+            String tagString = "PRESSED";
+            if (filterSection.getTag() == null) {
```

Comment:
Because of this https://github.com/OpenSRP/opensrp-client-child/blob/97abf1ccd7a24358a7b33901a92bdc3f7be8fab6/opensrp-child/src/main/java/org/smartregister/child/fragment/BaseChildRegisterFragment.java#L247

Reply:
But it can be changed

Reply:
Thanks, I also realized that. I am OK with whichever one you chose